ospo-book/chapters/03-chapter.md

```diff
@@ -1 +1,146 @@
+
+* **Management Level:** At this layer, the people behind the OSPO focus is on strategy and finding alignment between open source and the overall business / organizaiton strategy. Managers may face unique challenges, and leveraging the strengths of open source to help them solve these challenges can help address these issues effectively.
+
+* **Business / Team Units Level:** This layer focuses on how the people behind the OSPO ensures all the pieces of the organization structure fit together. It involves sharing best practices across different business / Team units and fostering collaboration and knowledge transfer among them.
+
+* **OSPO Level:** This is the inner side of the OSPO itself, where the people behind the OSPO works collectively to oversee and coordinate all the open source initiatives within the organization.
+
+This is a simplified view of the layers of work in an OSPO. However, depending on the complexity of your organization and the resources available to your OSPO, these layers can become more granular and include additional layers with different names.
+
+### The Pillars of Support
+
+![ospo-support](https://github.com/todogroup/ospology/assets/43671777/f96cd4a1-0315-4a0e-8de3-2da59378a57b)
+
+* Open Source Enabler: OSPOs can help organizations navigate the cultural, process, and tool changes required to engage with the open source community effectively. This can involve educating teams/ units, establishing new processes and workflows, and adopting new tools and technologies.
+
+* Open Source Counselor: OSPOs can provide guidance and advice on the latest open source trends, licensing issues, and how to engage with open source projects, foundations, and communities. This can help organizations stay up-to-date with the rapidly changing open source landscape and ensure they are making informed decisions.
+
+* Open Source Advocate: OSPOs can promote the use and/or contribution of open source and best practices across different organizational units. This can help organizations realize the benefits of open source as well as engaging people to contribute to open source projects or start new ones.
+
+* Open Source Environmentalist: OSPOs can help organizations support and sustain open source projects in the long term by addressing issues such as security, maintenance, and project health. This can involve establishing policies and procedures for code review, security vulnerability management, and ongoing maintenance and support through funding and/or contributions. By doing so, OSPOs can help ensure that open source projects remain healthy and continue to benefit the wider community.
+
+
+## Creating and Implementing an Open Source Strategy
+
+The way the people behind an OSPO achieve this is by creating and maintaining a framework covering the following aspects: strategy, governance, compliance, and community engagement. 
```

Comment:
```suggestion
The way the members of an OSPO achieve this is by creating and maintaining a framework covering the following aspects: strategy, governance, compliance, and community engagement. 
```

Reply:
I prefer using *people* since is the word we have used all the time. Let's discuss during contributor's call